### PR TITLE
Use paging to retreive alerts

### PR DESCRIPTION
**Pull request by psiinon**

### CODE REVIEW

File: build/docker/zap-baseline.py [hunk 1]
```diff
@@ -360,7 +360,23 @@ def main(argv):
...
+          if plugin_id in blacklist:
+            continue
+          if not is_in_scope(plugin_id, alert.get('url')):
+            continue
+          if (not alert_dict.has_key(plugin_id)):
+            alert_dict[plugin_id] = []
+          alert_dict[plugin_id].append(alert)
+        st += pg
+        alerts = zap.core.alerts(start=st, count=pg)
+
```
Comment (thc202) [inline build/docker/zap-baseline.py]: Some of the code below should be removed?

Comment (psiinon) [inline build/docker/zap-baseline.py]: Arg! cnp fail P

**Comment by psiinon** [comment]
This time tested the actual one in the PR rather than the one I copied from :P

**Comment by kingthorin** [comment]
Looks good

**Comment by thc202** [comment]
Thanks :)

**Pull request merged**